tests/BenchmarkDotNet.IntegrationTests.ManualRunning/MsBuildArgumentTests.cs

```diff
@@ -24,6 +24,13 @@ public void ProcessIsBuiltWithCustomProperty(bool setCustomProperty)
             CanExecute<PropertyDefine>(config);
         }
 
+        [Fact]
+
+        public void MsBuildProperty_ShouldWrapMultipleValuesInQuotes()
```

Comment:
Please move this test to `BenchmarkDotNet.Tests` project, and add a real integration test in `BenchmarkDotNet.IntegrationTests` project.